src/pivotal-ui/components/pui-variables.scss

```diff
@@ -461,8 +461,16 @@ $table-scrollable-body-border:   1px solid $gray-8;
 // Buttons
 // -------------------------
 
-$btn-font-weight:                600 !default;
+$btn-font-weight:                400 !default;
 $btn-border-width:               0;
+$btn-font-size:                  18px;
+
+$btn-sm-padding: 5px 20px 4px 20px;
```

Comment:
This was actually already the case, the variables were just in the buttons.scss file. I moved them over here. It's a good point though. I'll update the PR to make both of them vars.